### ISSUE TITLE
Enable spelling corrections via keyboard selection in the context menu

### ISSUE THREAD

**Issue by mjfs** [rstudio/rstudio#8833]
Spelling errors that are underlined in the document editor require selection among suggested corrections in the context menu only via mouse.

If RStudio would enable moving between suggested corrections in the context menu via keyboard that would enable faster editing. In addition, not relying on using the mouse is always a plus.

**Comment by ronblum** [comment]
@mjfs Thank you for the enhancement request! We'll review it as we continue development of RStudio.

**Comment by mjfs** [comment]
I can confirm that the issue remains unresolved in build 1.5.244.

The fact that one can not correct a spelling mistake in text based on spelling suggestion in context menu is really inconvenient.

This is such a common feature today that practically every application supporting text entry has it integrated, starting with simple text editors, IDEs, browsers, word processors etc.

In RStudio lack of this feature really hinders editing without a mouse. To add insult to injury, spellchecker window that pops up after a user explicitly runs "Check Spelling" (shortcut F7) also makes it difficult to not use a mouse since not even in that dialog there is a shortcut available (moving via TAB key is the only option).

**Comment by gtritchie** [comment]
Reopening. Ran across this doing an accessibility audit. This makes the spellchecking feature unusable via the keyboard.

**Comment by gtritchie** [comment]
When the context menu is opened, it doesn't receive keyboard focus, and thus cannot be operated via keyboard.

Note that on Windows you can show the context menu using standard Shift+F10 (or the dedicated context menu key on keyboard that have it), but that also doesn't put focus on the menu.

On macOS, you should be able to display the context menu using the "Full Keyboard Access" mode, then <kbd>Tab+M</kbd> but it displays the generic context menu, not the spell-check menu, so that also needs to be solved on macOS (in addition to the keyboard focus issue).
